app/views/articles/index.html.erb

```diff
@@ -1,11 +1,17 @@
 <h1>Articles</h1>
-
+Our blog has <%= Article.public_count %> articles and counting!
 <ul>
   <% @articles.each do |article| %>
+  <% unless article.archived? %>
     <li>
       <%= link_to article.title, article %>
     </li>
   <% end %>
+  <% end %>
 </ul>
 
 <%= link_to "New Article", new_article_path %>
+
+
```

Comment:
You should delete this unused lines.

Reply:
installed the rufus gem to improve code indentation